birdhouse/default.env

```diff
@@ -15,7 +15,21 @@ export DATA_PERSIST_ROOT="/data"
 # TODO: use this variable for other references of the log path (only used in the pavics-jupyter-base's .env file for now)
 export PAVICS_LOG_DIR=/var/log/PAVICS
 
-export DELAYED_EVAL=''
+# Allow different public and internal hostname.
+# Default to PAVICS_FQDN, which must be set in env.local.
+# Must use single-quote for delayed eval.
+export PAVICS_FQDN_PUBLIC='${PAVICS_FQDN}'
+
+# Ensure DELAYED_EVAL is initially empty.
+# Only this base default.env is allowed to reset this variable.  All component
+# default.env should only append to DELAYED_EVAL and not reset it like this.
+DELAYED_EVAL=''
+
+# Append to DELAYED_EVAL list.
+export DELAYED_EVAL="
+  $DELAYED_EVAL
+  PAVICS_FQDN_PUBLIC
+"
```

Comment:
I think a test would be in order here since it is not the "usual" `default.env` within a component/config sub-dir.

Reply:
Well before Misha's refactoring into components, delayed eval was used directly in the main `default.env` !

But maybe I should get acquainted to add test to Misha testsuite.